### DISCUSSION TITLE
Please make this marvel run well on a-Shell app for iPad

### DISCUSSION THREAD

**Discussion by Emasoft** [inducer/pudb#606] [Ideas]
**Is your feature request related to a problem? Please describe.**
[a-Shell](https://github.com/holzschu/a-shell) is an app for iPad that allows people to write, debug and run python and js code on the move like in a linux terminal. You go anywhere and you take your dev environment with you. And it is awesome. To avoid competition with ios apps, Apple limited to CUI the user interfaces of programs running inside a-Shell. So no fancy debuggers or ide. This is why your PUBD came in. Its literally the perfect companion for a-Shell. But it still need a little customization to run correctly. 

**Describe the solution you'd like**
Adapt PUBD to the a-Shell environment for iPad, until it would become the perfect tool for python development and debugging on iPad. There are some apps that sell themselves as Python IDEs for iOS, but none of those work well. They are too lacking, especially in the things PUBD excels. Also the CUI is great, because touch is all about big buttons, and big characters. Not to mention the fact that a-Shell can be configured to have thousand of characters wide terminal screen, in full true color!

Thanks! 🙏

**Comment by inducer** [comment]
I don't have Apple hardware available (nor any desire for it), so I'm not able to discern what doesn't yet "run well". To work as an issue with a definite resolution, I would need to know what (on a technical level) is amiss. Until then, I'm converting this to a discussion.
